docs/service-patterns/apply.md

```diff
@@ -0,0 +1,159 @@
+---
```

Comment:
Caption above the H1 for this page is 'Components'. Should this be 'Service patterns'?

![image](https://github.com/user-attachments/assets/15f785ad-8cca-4370-81a5-3e03d0051946)